c_cxx/QNN_EP/mobilenetv2_classification/run_qnn_ep_sample.bat

```diff
@@ -89,22 +87,24 @@ copy /y %ORT_BIN%\QnnHtpV68Stub.dll .
 copy /y %ORT_BIN%\QnnSystem.dll .
 copy /y %ORT_BIN%\libQnnHtpV68Skel.so .
```

Comment:
Per my [PR](https://github.com/microsoft/onnxruntime-inference-examples/pull/307) do we also want/need v73 from QNN SDK now? I am not sure myself but assume it's support for a newer chipset

copy /y %ORT_BIN%\QnnHtpV73Stub.dll
copy /y %ORT_BIN%\libQnnHtpV73Skel.so